rskj-core/src/main/java/co/rsk/peg/BridgeUtils.java

```diff
@@ -205,6 +206,17 @@ public static boolean isFromFederateMember(org.ethereum.core.Transaction rskTx,
         return federation.hasMemberWithRskAddress(rskTx.getSender().getBytes());
     }
 
+    public static Coin getCoinFromBigInteger(BigInteger value) throws BridgeIllegalArgumentException {
+        if (value == null) {
+            throw new BridgeIllegalArgumentException("value cannot be null");
+        }
+        try {
+            return Coin.valueOf(value.longValueExact());
+        } catch(ArithmeticException e) {
+            throw new BridgeIllegalArgumentException(e.getMessage());
```

Comment:
Why you pass only the message and leave the previous stack out of the exception?